train.py

```diff
@@ -1,209 +1,19 @@
+from model.model import RedshiftClassifierResNet, RedshiftClassifierInception
```

Comment:
NOTE: as most of the functionality here has been moved to model.py, this file was converted into a simple example script for how to use the redshift model module. I suspect that the name of the file is now inappropriate, and may need changing.